homeassistant/const.py

```diff
@@ -35,6 +35,7 @@
 CONF_AUTHENTICATION = "authentication"
 CONF_AUTH_MFA_MODULES = "auth_mfa_modules"
 CONF_AUTH_PROVIDERS = "auth_providers"
+CONF_AVAILABILITY_TEMPLATE = "availability_template"
```

Comment:
Move this into: /homassistant/components/template/const.py

Reply:
Done